src/seedu/addressbook/AddressBook.java

```diff
@@ -1150,12 +1150,12 @@ private static String getUsageInfoForExitCommand() {
     /**
      * Removes sign(p/, d/, etc) from parameter string
      *
-     * @param s  Parameter as a string
-     * @param sign  Parameter sign to be removed
+     * @param fullString  Parameter as a string
+     * @param prefix  Parameter sign to be removed
      * @return  string without the sign
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    private static String removePrefix(String fullString, String prefix) {
+        return fullString.replace(prefix, "");
```

Comment:
This method is not 'abstract' enough to remove ONLY the prefix. Hint: check for the index.

When you modify the behavior, remember to update documentation and tests as well.